addons/website_profile/i18n/website_profile.pot

```diff
@@ -423,13 +423,6 @@ msgstr ""
 msgid "No user found for"
 msgstr ""
 
-#. module: website_profile
-#. odoo-python
-#: code:addons/website_profile/controllers/main.py:0
-#, python-format
-msgid "Not have enough karma to view other users' profile."
```

Comment:
Note that we generally keep translations for people that do not update their source files

Reply:
> Note that we generally keep translations for people that do not update their source files

Hm, I'm not sure what you mean since if I change the source term (i.e. the string in python) then there will be nothing to match the translation to. Either way though, I can also do this fix only in master if you prefer since it's not blocking, it's just a bit Yoda-y.

Reply:
If people don't pull the python sources, they keep and use the old string, so it makes sense to keep it in the pot file in stable so that people translate it. That is why we generally keep old sentences in stable (when there is no uniqueness constraint, we can't do that for translations linked to records).

Reply:
If people don't pull the python sources, they keep and use the old string, so it makes sense to keep it in the pot file in stable so that people translate it. That is why we generally keep old sentences in stable (when there is no uniqueness constraint, we can't do that for translations linked to records).